modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +191,35 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+A Mule application can contain several batch job definitions. Each definition 
+has its own scheduling strategy. The Batch Job scope provides 
+*Scheduling Strategy* options for controlling the execution of job instances: 
+
+* ORDERED_SQUENTIAL (the default): If several job instances are in an executable
+state at the same time, the instances execute one at a time based on their
+creation timestamp.
+* ROUND_ROBIN: This setting attempts to execute all available instances of a
+batch job using a round-robin algorithm to assign the available resources. 
++
+This option is useful when you can guarantee that no job execution can have a 
+side effect on another job execution. Therefore, this option is not a good 
+choice for data synchronization jobs, which can update the same record in two 
+concurrent jobs. Because the order of the job execution is not guaranteed, 
+you might end up with a prior version of the data. However, you can safely 
+use this strategy to parallelize the job's execution if your batch job 
+retrieves only new records from a database or lifts individual files 
+from an SFTP server (and you are certain that all records are completely 
+independent).
++
+This setting was introduced before the current default, ORDERED_SQUENTIAL.
+
+Note that there is no guarantee that records will be executed in order.
+The strategies do not control the execution order of records in the batch job,
+nor do they take into account the number of records each instance contains.
```

Comment:
`nor do they take into account the number of records each instance contains.` -> `nor do they depend on the  number of records each instance contains.`

Reply:
done